aptos-move/framework/aptos-framework/sources/configs/stake.move

```diff
@@ -650,37 +641,33 @@ module aptos_framework::stake {
     }
 
     /// Similar to increase_lockup_with_cap but will use ownership capability from the signing account.
-    public entry fun increase_lockup(
-        account: &signer,
-        new_locked_until_secs: u64,
-    ) acquires OwnerCapability, StakePool, StakePoolEvents, ValidatorSetConfiguration {
+    public entry fun increase_lockup(account: &signer)
```

Comment:
do we still need an entry function for it? I thought it's going to be recurring?

Reply:
This is for "resetting" the lockup that's needed for governance. It's the last point in the flow mentioned in the description:

6. 25 days into the current one month cycle, a governance proposal is submitted with a voting duration of 7 days. Alice only has 5 days left in her current lockup. This is not enough to guarantee her stake will stay locked for the 7-day voting period. Alice wants to participate in voting on this proposal so she resets her lockup, which locks her stake for 30 days. She can now vote on the governance proposal.